File: problem2/partitionsave.py
rdd = sc.textFile("/user/training/problem1/output")
rddDropped = rdd.filter(lambda line: line.split("\t")[3] == "DROPPED")
rddSuccess = rdd.filter(lambda line: line.split("\t")[3] == "SUCCESS")
rddFailed = rdd.filter(lambda line: line.split("\t")[3] == "FAILED")

rddDropped.saveAsTextFile("/user/training/problem2/output/status=DROPPED")
rddFailed.saveAsTextFile("/user/training/problem2/output/status=FAILED")
rddSuccess.saveAsTextFile("/user/training/problem2/output/status=SUCCESS")

File: problem2/partitionsave.scala
val rdd = sc.textFile("/user/training/problem1/output")
	val rddDrop = rdd.filter(line => line.contains("DROPPED")
	val rddFail = rdd.filter(line => line.contains("FAILED")
	val rddSucc = rdd.filter(line => line.contains("SUCCESS")

	rddDrop.saveAsTextFile("/user/training/problem2/output/status=DROPPED")
	rddFail.saveAsTextFile("/user/training/problem2/output/status=FAILED")
	rddSucc.saveAsTextFile("/user/training/problem2/output/status=SUCCESS")
